src/rtp/vla.rs

```diff
@@ -261,6 +389,21 @@ fn parse_leb_u63(bytes: &[u8]) -> (u64, &[u8]) {
     (0, bytes)
 }
 
+/// Encodes leb128
+pub fn encode_leb_u64(mut value: u64) -> Vec<u8> {
```

Comment:
Ya - I just made no attempt to prevent a full 64bit value being encoded so just named it.. I also noticed the asymmetry but it wouldn't actually being u63 for the encode side

Reply:
Ya - I just made no attempt to prevent a full 64bit value being encoded so just named it.. I also noticed the asymmetry but it wouldn't actually being u63 for the encode side

Reply:
Cool. Maybe it doesn't matter because no one is encoding full 64 bits with it anyway?

Reply:
Renamed and made to use underlying buf instead of vec